hypershift-operator/controllers/hostedcluster/internal/platform/platform.go

```diff
@@ -27,11 +28,7 @@ type Platform interface {
 	// Implementations should use the given input and client to create and update the desired state of the
 	// platform infrastructure CAPI CR, which will then be referenced by the CAPI Cluster CR.
 	// TODO (alberto): Pass createOrUpdate construct instead of client.
-	ReconcileCAPIInfraCR(ctx context.Context, c client.Client, createOrUpdate upsert.CreateOrUpdateFN,
-		hcluster *hyperv1.HostedCluster,
-		controlPlaneNamespace string,
-		apiEndpoint hyperv1.APIEndpoint,
-	) (client.Object, error)
+	ReconcileCAPIInfraCR(ctx context.Context, c client.Client, createOrUpdate upsert.CreateOrUpdateFN, hcluster *hyperv1.HostedCluster, controlPlaneNamespace string, apiEndpoint hyperv1.APIEndpoint) (client.Object, error)
```

Comment:
this is not changing the signature but just inlining everything, right?
fwiw I broke it up this way so lines beyond ~110 get difficult to read in 13-inch. I'm fine with the change.

Reply:
Yeah it just changes the style, I did that because I find signatures that have multiple but not all parameters on a single line confusing to read, IMHO either all should be on one line like right now or each one gets their own line